src/box/space.c

```diff
@@ -56,6 +56,7 @@
 #include "wal_ext.h"
 #include "coll_id_cache.h"
 #include "func_adapter.h"
+#include "lua/utils.h"
```

Comment:
The only thing I don't like is that dependency. Perhaps there's a way to avoid it>

Reply:
Discussed offline, it's decided this is the lesser evil.